public/waitingroom.css

```
@@ -0,0 +1,632 @@
+/* ================================================================
+   waitingroom.css — Shared styles for waiting room, courses, and
+   teach-course pages (nav, page layout, cards, modal, footer).
+   ================================================================ */
+
+/* ---------- Reset & Base ---------- */
+*, *::before, *::after { box-sizing: border-box; }
+body {
+  margin: 0;
+  font-family: 'Inter', system-ui, -apple-system, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)